Mathlib/Algebra/Module/LinearMap/Rat.lean

```diff
@@ -30,3 +30,13 @@ theorem AddMonoidHom.toRatLinearMap_injective [AddCommGroup M] [Module ℚ M] [A
 theorem AddMonoidHom.coe_toRatLinearMap [AddCommGroup M] [Module ℚ M] [AddCommGroup M₂]
     [Module ℚ M₂] (f : M →+ M₂) : ⇑f.toRatLinearMap = f :=
   rfl
+
+instance {R M N : Type*} [Ring R] [AddCommGroup M] [AddCommGroup N]
+    [Module R M] [Module R N] [Module ℚ M] [Module ℚ N] :
+    MulActionHomClass (M →ₗ[R] N) ℚ M N where
```

Comment:
I don't think this instance is safe, since the second argument is an outParam, and could equally be chosen as `R`.

Reply:
Oooh you're right. Also it's redundant: this is a job for `map_rat_smul`.